2017-05-01 19:27:04.801 [error] <0.706.0> Supervisor 'Elixir.Logger.Supervisor' had child undefined started with 'Elixir.Logger.Watcher':watcher(error_logger, 'Elixir.Logger.ErrorHandler', {true,false,500}) at <0.711.0> exit with reason normal in context child_terminated
2017-05-01 19:27:04.832 [error] <0.749.0> CRASH REPORT Process <0.749.0> with 0 neighbours exited with reason: no match of right hand value undefined in mcd_cluster_sup:buckets/0 line 60 in gen_server:init_it/6 line 352
2017-05-01 19:27:04.832 [error] <0.747.0> CRASH REPORT Process <0.747.0> with 0 neighbours exited with reason: {{{badmatch,undefined},[{mcd_cluster_sup,buckets,0,[{file,"/Users/enrique/webs/memcachir/deps/mcd/src/mcd_cluster_sup.erl"},{line,60}]},{mcd_cluster_sup,clusters,0,[{file,"/Users/enrique/webs/memcachir/deps/mcd/src/mcd_cluster_sup.erl"},{line,64}]},{mcd_cluster_sup,init,1,[{file,"/Users/enrique/webs/memcachir/deps/mcd/src/mcd_cluster_sup.erl"},{line,41}]},{supervisor,init,1,[{file,"supervisor.erl"},{line,294}]},{gen_server,init_it,6,[{file,"gen_server.erl"},{line,328}]},{proc_lib,init_p_do_apply,...}]},...} in application_master:init/4 line 134
2017-05-01 19:28:57.187 [error] <0.156.0> Supervisor 'Elixir.Logger.Supervisor' had child undefined started with 'Elixir.Logger.Watcher':watcher(error_logger, 'Elixir.Logger.ErrorHandler', {true,false,500}) at <0.161.0> exit with reason normal in context child_terminated
2017-05-01 19:28:57.215 [error] <0.199.0> CRASH REPORT Process <0.199.0> with 0 neighbours exited with reason: no match of right hand value undefined in mcd_cluster_sup:buckets/0 line 60 in gen_server:init_it/6 line 352
2017-05-01 19:30:52.893 [error] <0.210.0> Supervisor 'Elixir.Logger.Supervisor' had child undefined started with 'Elixir.Logger.Watcher':watcher(error_logger, 'Elixir.Logger.ErrorHandler', {true,false,500}) at <0.215.0> exit with reason normal in context child_terminated
2017-05-01 19:30:52.929 [error] <0.253.0> CRASH REPORT Process <0.253.0> with 0 neighbours exited with reason: no match of right hand value undefined in mcd_cluster_sup:buckets/0 line 60 in gen_server:init_it/6 line 352
2017-05-01 19:30:52.929 [error] <0.251.0> CRASH REPORT Process <0.251.0> with 0 neighbours exited with reason: {{{badmatch,undefined},[{mcd_cluster_sup,buckets,0,[{file,"/Users/enrique/webs/memcachir/deps/mcd/src/mcd_cluster_sup.erl"},{line,60}]},{mcd_cluster_sup,clusters,0,[{file,"/Users/enrique/webs/memcachir/deps/mcd/src/mcd_cluster_sup.erl"},{line,64}]},{mcd_cluster_sup,init,1,[{file,"/Users/enrique/webs/memcachir/deps/mcd/src/mcd_cluster_sup.erl"},{line,41}]},{supervisor,init,1,[{file,"supervisor.erl"},{line,294}]},{gen_server,init_it,6,[{file,"gen_server.erl"},{line,328}]},{proc_lib,init_p_do_apply,...}]},...} in application_master:init/4 line 134
